src/Conf.cpp

```diff
@@ -32,6 +33,22 @@ bool ServerConf::parse(ServerConf& s, const char* str)
     return !s.addr.empty();
 }
 
+void CustomCommandConf::init(CustomCommandConf&c, const char* name, const int type) {
+    c.name = name;
+    c.type = type;
+    c.minArgs = 2;
+    c.maxArgs = 2;
+    c.mode = Command::Write;
+}
+
+void CustomCommandConf::convert(const CustomCommandConf&c, Command &cmd) {
+    cmd.name = strdup(c.name.c_str());
```

Comment:
**strdup** will alloc memory, although memory leak is no matter, but we should avoid to do this. see other comment.